en/sensor/cm8jl65_ir_distance_sensor.md

```diff
@@ -1,4 +1,10 @@
-# Lanbao PSK-CM8JL65-CC5 ToF Infrared Distance Measuring Sensor
+# Lanbao PSK-CM8JL65-CC5 ToF Infrared Distance Measuring Sensor (Discontinued)
+
+<Badge type="info" text="Discontinued" />
+
+:::warning
+This product has been [discontinued](../flight_controller/autopilot_experimental.md) and is no longer commercially available.
```

Comment:
@hamishwillee 
You link here to the "Autopilot Experimental" Page, is this intended?
As there is no info in regard of discontinued products on that page.

Reply:
No, that's a copy-paste error. Will fix now. Good catch.

Reply:
Fixed in https://github.com/PX4/PX4-user_guide/pull/3498